package.json

```diff
@@ -14,8 +14,10 @@
     "to-commonjs": "node build-tools/replace_package_type.cjs module commonjs",
     "build-vercel-pkg": "node web-local/dist/web-local/src/cli/build-vercel-pkg.js",
     "to-esm": "node build-tools/replace_package_type.cjs commonjs module",
-    "dev": "npm run dev -w web-local",
-    "dev-runtime": "go run cli/main.go start --no-ui",
+    "dev": "sh -c 'npm run dev-runtime & npm run dev-web & wait'",
```

Comment:
We need a way to pass in `--dir` down to dev-runtime somehow. Not a blocker for this PR